numpy/core/setup_common.py

```diff
@@ -124,9 +124,8 @@ def set_sig(sig):
         "floor", "ceil", "sqrt", "log10", "log", "exp", "asin",
         "acos", "atan", "fmod", 'modf', 'frexp', 'ldexp']
 
-# Standard functions which may not be available and for which we have a
-# replacement implementation. Note that some of these are C99 functions.
-OPTIONAL_STDFUNCS = ["expm1", "log1p", "acosh", "asinh", "atanh",
+# Some of these are C99 functions.
```

Comment:
As a further cleanup, consider merging the elements of this list into the list right above, and remove `MANDATORY_FUNCS +=`

Reply:
done